code/modules/mob/living/emote.dm

```diff
@@ -263,7 +263,9 @@
 		return
 	else
 		var/sound
-		if(!user.is_muzzled() && !user.mind.miming)
+		var/miming = user.mind ? user.mind.miming : 0

```

Comment:
Good idea, but gonna leave that out of this PR as this is just for bugfixing embedded objects